source/iostreams.tex

```diff
@@ -11861,21 +11861,23 @@
 \begin{example}
 Even if \tcode{//host} is interpreted as a \grammarterm{root-name},
 both of the paths \tcode{path("//host")/"foo"} and \tcode{path("//host/")/"foo"}
-equal \tcode{"//host/foo"}.
+equal \tcode{"//host/foo"} (although the former might use backslash as the
+preferred separator).
 
 Expression examples:
 \begin{codeblock}
 // On POSIX,
-path("foo") / "";     // yields \tcode{"foo/"}
-path("foo") / "/bar"; // yields \tcode{"/bar"}
-// On Windows, backslashes replace slashes in the above yields
+path("foo") /= path("");        // yields \tcode{path("foo/")}
+path("foo") /= path("/bar");    // yields \tcode{path("/bar")}
 
 // On Windows,
-path("foo") / "c:/bar";  // yields \tcode{"c:/bar"}
-path("foo") / "c:";      // yields \tcode{"c:"}
-path("c:") / "";         // yields \tcode{"c:"}
-path("c:foo") / "/bar";  // yields \tcode{"c:/bar"}
-path("c:foo") / "c:bar"; // yields \tcode{"c:foo/bar"}
+path("foo") /= path("");        // yields \tcode{path("foo\textbackslash{}")}
```

Comment:
Don't we need two consecutive backslashes in something claiming to be a C string?

Reply:
Indeed we do, thanks. I've pushed a fixed and rebased update.